nncf/quantization/layers.py

```diff
@@ -191,7 +194,53 @@ def set_export_mode(self, mode: QuantizerExportMode):
         self._export_mode = mode
 
     def run_export_quantization(self, x: torch.Tensor):
-        raise NotImplementedError
+        with no_jit_trace():
+            if isinstance(self, SymmetricQuantizer):
+                input_range = abs(self.scale) + self.eps
+                input_low = input_range * self.level_low / self.level_high
+                input_high = input_range
+            elif isinstance(self, AsymmetricQuantizer):
+                input_range_safe = abs(self.input_range) + self.eps
+                input_low, input_range_tuned = TuneRange.apply(self.input_low, input_range_safe, self.levels)
+                input_high = input_low + input_range_tuned
+            else:
+                raise NotImplementedError
```

Comment:
... and keep these in respective subclasses. Make an abstract `_get_input_low_input_high` method in `BaseQuantizer`, implement in specific subclasses accordingly. In `BaseQuantizer.run_export_quantization`, call this method.